src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+    private static final Map<String, Integer> STORAGE = FruitStorage.fruits;
```

Comment:
you can use FruitStorage.fruits directly. You don't need to create additional variable